Barlist/Report.cpp

```diff
@@ -106,7 +106,10 @@ void CReport::BuildReport(IBarlist* pBarlist)
    m_vReportLines.reserve(1000);
 
    ReportGroups(pBarlist);
-   //ReportQuantitiesByGroup(pBarlist);
+   if (reportOptions == REPORT_TOTAL_AND_GROUP_QUANTITIES) 
```

Comment:
what happens if reportOptions == REPORT_GROUP_QUANTITIES?
I think ReportOptions should be limited to two choices - see comment below.

Reply:
Ok